src/platform/forms/save-in-progress/SaveFormLink.jsx

```diff
@@ -42,7 +43,11 @@ class SaveFormLink extends React.Component {
     const appType = formConfig?.customText?.appType || APP_TYPE_DEFAULT;
 
     return (
-      <div className={this.props.children ? 'vads-u-display--inline' : ''}>
+      <div
+        className={`${
+          this.props.children ? 'vads-u-display--inline' : ''
+        }vads-u-margin-top--2`}
```

Comment:
It looks like we could end up with something like `vads-u-display--inlinevads-u-margin-top--2` here if I'm reading it correctly. Also, is this necessary if we're already adding vertical margins to the `FormNavButtons`?

Reply:
Oops, you're right... I'll fix this in the morning. Huh, I thought that was adding a right margin. I'll fix that too.

Reply:
I ended up reverting this margin-top addition. The form wrapper above it already includes a `vads-u-margin-y--2` and this addition didn't add any more spacing. We don't need to update the `FormNavButton` margins or the SiP menu left margin.